src/lib/storage/boot_requirements_strategies/legacy.rb

```diff
@@ -29,45 +29,45 @@ module BootRequirementsStrategies
       # Strategy to calculate the boot requirements in a legacy system (x86
       # without EFI)
       class Legacy < Base
+        GRUB_SIZE = DiskSize.KiB(256)
+        GRUBENV_SIZE = DiskSize.KiB(2)
```

Comment:
Well, I wanted to stay safe for future. We also have 256 kib for grub even it is currently ~85 kib. We could revisit both values in the future, after the initial mchang's experiments of the new approach.